guardian/support-admin-console
# Add newsletter sign up to Live Blog Epics

## What does this change?
Adds newsletter sign up component to the live blog epics. It introduces new behaviour which means that you can only have one of the two ctas either buttons and choice cards or newsletter sign ups. [Related to this PR in DCR](https://github.com/guardian/dotcom-rendering/pull/11649).


## How to test
On the RRCP create Live Blog Epic  test and that you can create a variant with buttons and another with the newsletter sign up. View variant on 'web preview' and see the epic with the newsletter sign up.

Tested on CODE following the above steps. Expected behaviour on desktop and mobile (via browserstack) and on chrome, firefox, safari and edge.


## How can we measure success?



## Have we considered potential risks?



## Images
On Buttons selection
<img width="1125" alt="image" src="https://github.com/guardian/support-admin-console/assets/115992455/0cf9e659-43ad-4e38-b8a3-81347f284c5a">

On Newsletter signup
<img width="1140" alt="image" src="https://github.com/guardian/support-admin-console/assets/115992455/74352a19-d89d-4d71-a35a-cfabebd8671f">

On CODE via web preview
![image](https://github.com/guardian/support-admin-console/assets/115992455/9c393e24-50a2-43aa-94e3-da9ed3335550)



## Accessibility
